apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -1,10 +1,14 @@
 import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
 import color from "@cdo/apps/util/color";
 import i18n from '@cdo/locale';
 import Button from '../Button';
+import TeacherInfoBox from '@cdo/apps/templates/progress/TeacherInfoBox';
+import { ViewType } from '@cdo/apps/code-studio/stageLockRedux';
 
 const styles = {
   main: {
+    display: 'table',
```

Comment:
What's this required to do?

Reply:
I think it's needed to make the TeacherInfoBox display correctly, since it uses display: table-cell